admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1082,9 +1082,12 @@ $product-page-padding-bottom: 80px !default;
     }
   }
 
-  #product_stock_quantities_stock_movements {
-    max-width: 400px;
+  #product_stock_quantities_stock_movement_histories {
+    max-width: 600px;
+  }
 
+  #product_stock_quantities_stock_movement_histories,
+  #combination_form_stock_quantities_stock_movement_histories {
```

Comment:
Should I split the combination product CSS into a distinct file ?

Reply:
I don't think so but to be sure ⬇️ 
ping @jolelievre and @NeOMakinG

Reply:
I would say no, since there are no much things related to it in the main file, until the CSS gets big